config/default-spinnaker-local.yml

```diff
@@ -121,6 +121,21 @@ providers:
       # A path to a plain text file containing the user's password
       passwordFile: ${SPINNAKER_DOCKER_PASSWORD_FILE}
 
+  openstack:
+    # For more information on configuring Openstack clusters, see
+    # TODO add link to docs when available
+
+    enabled: ${SPINNAKER_OPENSTACK_ENABLED:false}
+    primaryCredentials:
+      name: my-openstack-account
+      authUrl: ${SPINNAKER_OPENSTACK_AUTH_URL}
+      username: ${SPINNAKER_OPENSTACK_USERNAME}
+      password: ${SPINNAKER_OPENSTACK_PASSWORD}
```

Comment:
ok. there's precedent for them being here (e.g. jenkins). FYI azure and cf had started out with them here but moved them out. They might have added mechanisms for it or maybe their platforms had them and they werent using them initially. Having passwords here makes it easy to forget and do something like check the file into a revision control system or forget to chmod 600 (or 400) the file.